src/main/webapp/scripts/session-script.js

```diff
@@ -1,46 +1,84 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
+ * An array of who is currently in the session.
+ * @type {Object}
```

Comment:
Array, not Object (even though technically everything in JS in an object, but still. Array is the concrete type here). Fix this everywhere its a mistake too